app/javascript/components/common/ProgressGraph.tsx

```diff
@@ -57,6 +49,15 @@ export const ProgressGraph: React.FC<IProgressGraph> = ({
       }`}
     >
       <defs>
+        <mask
+          id={`progress-graph-color-mask-${getRandomId()}`}
+          x="0"
+          y="0"
+          width={width}
+          height={height}
+        >
+          <path d={path} stroke="white" fill="transparent" />
+        </mask>
```

Comment:
As it turns out, when a vertical gradient is applied to a horizontal (flat) line, the svg breaks and shows nothing.  This method is that the line creates a mask over a rectangle that is colored with the gradient, and the mask only shows the resulting line and thus works for horizontal lines.